psp-examples/.gitignore

```diff
@@ -19,3 +19,4 @@ test-ledger
 
 build-circuit
 **/verifying_key_*.rs
+**/*.circom
```

Comment:
This is too wide ranging.
We need to keep the circom files in the private compressed account example.

Reply:
Then should we keep .gitignore files per PSP project?

Reply:
```suggestion
**/*.circom
!**/private-compressed-account/**/*.circom
```